src/Microsoft.Azure.WebJobs.Host/Extensions/IConfigurationExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host;
+
+namespace Microsoft.Extensions.Configuration
+{
+    public static class IConfigurationExtensions
+    {
+        public static string GetWebJobsConnectionString(this IConfiguration configuration, string connectionStringName)
```

Comment:
Chose to migrate our current prefixing logic into an extension method.

Reply:
Love it.

Reply:
This does work a little different than what we had discussed, right?

This will expect connection strings to be defined in the root, under the connection strings section, as opposed to being in the extension scope/configuration section.

With this approach, are we proposing that all connection strings go under the root? We need to settle on this to avoid variation (which we have a bit of).

Reply:
This method just takes an IConfiguration which can be an extension config section. So the caller determines the scope. This method then checks connection strings then direct config for the value.

I did think about extensions with these changes, and wondered if I needed to add an overload taking an extension section name. However, all tests pass with these changes - it doesn't look like any extensions require anything else, so I didn't add anything else.

I'm in the process of pulling these updates into Extensions now and will verify everything works.

Reply:
Extensions all work with these changes - see PR https://github.com/Azure/azure-webjobs-sdk-extensions/pull/458.